pkg/deploy/webhook_chart.go

```diff
@@ -1,61 +1,122 @@
 package deploy
 
 import (
+	"bytes"
 	"context"
 	"errors"
 	"fmt"
 
+	"github.com/kyma-project/lifecycle-manager/api/v1alpha1"
+	admissionv1 "k8s.io/api/admissionregistration/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	modulelabels "github.com/kyma-project/module-manager/operator/pkg/labels"
+	moduletypes "github.com/kyma-project/module-manager/operator/pkg/types"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
 	"k8s.io/client-go/rest"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-
-	moduleLib "github.com/kyma-project/module-manager/operator/pkg/manifest"
-	moduleLibTypes "github.com/kyma-project/module-manager/operator/pkg/types"
 )
 
 type Mode string
 
 const (
-	ModeInstall            = Mode("install")
-	ModeUninstall          = Mode("uninstall")
-	customConfigKey        = "modules"
-	ReleaseName            = "skr"
-	IstioSytemNs           = "istio-system"
-	IngressServiceName     = "istio-ingressgateway"
-	DeploymentNameTpl      = "%s-webhook"
-	triggerLabelTimeFormat = "200601021504050700"
+	customConfigKey         = "modules"
+	IstioSytemNs            = "istio-system"
+	IngressServiceName      = "istio-ingressgateway"
```

Comment:
I still think this should be configurable in the controller via flag, same as the namespace. An Istio-installation can look different than this and it heavily opinionated